src/methods/defining_methods/hard/q5/FavMovie.java

```diff
@@ -0,0 +1,39 @@
+package methods.defining_methods.hard.q5;
+
+/**
+ * [J] Write a Java Program that asks users for their favourite movie and asks them how many times they've watched that movie. Then,
+ * write a Java Method that takes the string and integer as parameters and returns a string. The returned string should be as follows.
+ * If the integer is 5 or lower, then just return a string saying: "You've watched <the movie name> a few times!", if the integer is
+ * smaller than 10 and smaller than 5, then return a string saying: "You've watched <the movie name> a lot of times!", if the integer
+ * is greater than 10, then return a string saying: "You've watched <the movie name> so many times! Wow!". Call your method movieCounter
+ * and pass in the movie name and the number of times you have watched the movie respectively.
```

Comment:
We should also probably specify what to do with negative numbers because they don't make sense in this context